packages/web/src/components/list/SingleDropdownList.js

```diff
@@ -108,12 +108,24 @@ class SingleDropdownList extends Component {
 
 	defaultQuery = (value, props) => {
 		if (this.props.selectAllLabel && this.props.selectAllLabel === value) {
+			if (this.props.showMissing === true) {
+				return {};
```

Comment:
I think it would be better if we return a `{ match_all: {} }` query here, instead of `{}`, allowing us to clearly state the intent that we expect all the results from this component's query.

Also, notice how we pass `props` as a param to this method. So, you should rewrite the boolean condition here as:

```
if (props.showMissing)
```